go.mod

```diff
@@ -3,9 +3,14 @@ module github.com/ONSdigital/dp-topic-api
 go 1.15
 
 require (
+	github.com/ONSdigital/dp-api-clients-go v1.28.0
+	github.com/ONSdigital/dp-authorisation v0.1.0
 	github.com/ONSdigital/dp-healthcheck v1.0.5
-	github.com/ONSdigital/dp-net v1.0.7
+	github.com/ONSdigital/dp-image-api v0.10.0
```

Comment:
I think this dependency is not needed. 
Can you run `go mod tidy`, and push the changes, please?

Reply:
done